### ISSUE TITLE
Patches to contact_storage_export don't apply

### ISSUE THREAD

**Issue by giucas-falchetto** [openeuropa/oe_contact_forms#78]
The two patches are not required for the latest version (8.x-1.15) of contact_storage_export module.
```
- Applying patches for drupal/contact_storage_export
    https://www.drupal.org/files/issues/2021-10-22/contact_storage_export-breaks-on-high-load-balancer-2996037-17.patch (https://www.drupal.org/project/contact_storage_export/issues/2996037)
   Could not apply patch! Skipping. The error was: Cannot apply patch https://www.drupal.org/files/issues/2021-10-22/contact_storage_export-breaks-on-high-load-balancer-2996037-17.patch
    https://www.drupal.org/files/issues/2020-04-14/php73-3049010-15.patch (https://www.drupal.org/project/contact_storage_export/issues/3049010)
   Could not apply patch! Skipping. The error was: Cannot apply patch https://www.drupa
```

**Comment by brummbar** [comment]
Hello @giucas-falchetto , we will fix this asap. Thanks for reporting.

**Comment by brummbar** [comment]
In the meanwhile, you can [exclude the patches in your composer.json ](https://github.com/vaimo/composer-patches/blob/master/docs/USAGE_ADVANCED.md#exclusion).

**Comment by brummbar** [comment]
It was actually already fixed in 1.10.0.
